ivadomed/transforms.py

```diff
@@ -296,3 +301,126 @@ def __call__(self, sample):
             sample.update(rdict)
 
         return sample
+
+
+class StackTensors(mt_transforms.StackTensors):
+    """This class extends mt_transforms.NormalizeInstance"""
+    def undo_transform(self, sample):
+        rdict = {}
```

Comment:
No update of the Tensor? I am not familiar with this function, trust you ;-)

Reply:
Under undo transforms don't support, undo on lists of tensors so keeping it concatenate is more useful. For now, I see no benefit in resplitting the input in lists of tensors.